src/app/services/env.service.ts

```diff
@@ -7,7 +7,9 @@ export class EnvService {
   // API_URL = 'http://localhost:8003/api.php?action=';
   // API_URL = 'http://localhost/AltaraCredit/altara-api/api.php?action=';
   API_URL = 'https://altara-api.herokuapp.com/api.php?action=';
-  NEW_API_URL="https://altara-play-api.herokuapp.com"
+  LIVE_NEW_API_URL = "https://altara-prod-api.herokuapp.com"

```

Comment:
How are you resolving this two endpoints???

Reply:
I switch them based on the build